package.json

```diff
@@ -26,8 +26,11 @@
   },
   "dependencies": {
     "@sphereon/ssi-types": "0.8.1-next.31",
+    "@types/jwt-encode": "^1.0.0",
```

Comment:
Why are we adding @types as dep and not as devDep? It is only needed for typing

Reply:
removed this and used jose